cmd/juju/cloud/list.go

```diff
@@ -180,34 +178,31 @@ func formatCloudsTabular(writer io.Writer, value interface{}) error {
 		return names
 	}
 
-	printClouds := func(someClouds map[string]*cloudDetails) {
+	printClouds := func(someClouds map[string]*cloudDetails, colour *ansiterm.Context) {
```

Comment:
But, but, but.... it grates so much to look at the wrong spelling. Sigh. Changed.